synapse_invite_checker/invite_checker.py

```diff
@@ -801,6 +810,55 @@ async def get_delete_tasks_by_room(self, room_id: str) -> list[ScheduledTask]:
             statuses=statuses,
         )
 
+    async def kick_all_local_users(self, room_id: str) -> None:
+        """
+        First block the room from further interaction from this server, then cause all
+        local members to set their membership to 'leave'
+
+        Args:
+            room_id:
+        """
+        requester_user_id = f"@auto-kicker:{self.api._hs.config.server.server_name}"
```

Comment:
```suggestion
        requester_user_id = f"@_synapse-invite-checker:{self.api._hs.config.server.server_name}"
```

Underscore is a reserved namespace afaik and I think it makes sense to use the module name here?

Reply:
It gets written to a database table that is never looked at again, and no processing is done on it :laughing: I changed it to this anyway

Reply:
But it would be weird to use a registerable userid, imo :)

Reply:
Also makes debugging easier!